src/imars3d/backend/dataio/data.py

```diff
@@ -475,41 +475,131 @@ def _extract_rotation_angles(
     return rotation_angles
 
 
+def _to_time_str(value: datetime) -> str:
+    """
+    Convert the supplied datetime to a formatted string.
+
+    Parameters
+    ----------
+    value:
+        datetime object to format correctly
+
+    Returns
+    -------
+        The datetime as YYYYMMDDhhmm
+    """
+    return value.strftime("%Y%m%d%H%M")
+
+
+def _save_data(filename: Path, data: np.ndarray, omegas: np.ndarray = None) -> None:
```

Comment:
quick question: is it necessary to specify `-> None` for a function with no return?